wakuv2/waku.go

```diff
@@ -79,6 +78,7 @@ import (
 
 const messageQueueLimit = 1024
 const requestTimeout = 30 * time.Second
+const dialTimeout = 30 * time.Second
```

Comment:
Wondering if 30seconds is too long a dialTimeout.

Reply:
Good question.
The default from go-libp2p is 60s
And the value it had earlier was 7s which seemed to little IMO.

Reply:
This is no longer an issue, because the latest commit uses the AddDiscoveredPeer API!